conf/standalone.conf

```diff
@@ -81,6 +81,17 @@ maxUnackedMessagesPerConsumer=50000
 # check and dispatcher can dispatch messages without any restriction
 maxUnackedMessagesPerSubscription=200000
 
+# Max number of unacknowledged messages allowed per broker. Once this limit reaches, broker will stop dispatching
+# messages to all shared subscription which has higher number of unack messages until subscriptions start
+# acknowledging messages back and unack count reaches to limit/2. Using a value of 0, is disabling
+# unackedMessage-limit check and broker doesn't block dispatchers
+maxUnackedMessagesPerBroker=20000000
```

Comment:
actually even for other configurations we have maintained same value in all config files. Isn't 2m will be low number, instead can we keep 10m.? I will change the value.